eng/Versions.props

```diff
@@ -6,7 +6,7 @@
     <MajorVersion>6</MajorVersion>
     <MinorVersion>0</MinorVersion>
     <PatchVersion>3</PatchVersion>
-    <SdkBandVersion>6.0.100</SdkBandVersion>
+    <SdkBandVersion>6.0.200</SdkBandVersion>
```

Comment:
should this be removed now?

Reply:
No, it's still needed by the wasm tests.  We can fix that up in another PR.